### PR TITLE
Add versioning to docker-storage-setup.

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <shishir.mahajan@redhat.com>

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
ping @rhvgoyal

**Comment by rhvgoyal** [comment]
@shishir-a412ed can you update this PR please.

File: docker-storage-setup.sh [hunk 1]
```diff
@@ -1167,6 +1167,10 @@ reset_storage() {
   rm -f $DOCKER_STORAGE
 }
 
+print_version() {
+  echo "1.0.0"
```
Comment (rhvgoyal) [inline docker-storage-setup.sh]: Lets store version information in a variable and echo that variable.

**Comment by shishir-a412ed** [comment]
@rhvgoyal PTAL.

**Comment by rhvgoyal** [comment]
@shishir-a412ed So how about following.

MAJOR_VERSION=0
MINOR_VERSION=1
SUBLEVEL=0
EXTRA_VERSION=""

And final version will be
VERSION=${MAJOR_VERSION}.${MINOR_VERSION}.${SUBLEVEL}
[ -n "$EXTRA_VERSION" ] && VERSION="${VERSION}-$EXTRA_VERSION"

That is initially our version will look like.

0.1.0

Once we have something stable we will bump it to.

1.0.0

And after that during development, we can add extra version like -dev, or -rc1 etc.

1.0.0-dev
1.0.0-rc1

and then release a new version say
1.1.0

@rhatdan WDYT about above versioning scheme for docker-storage-setup.

**Comment by rhatdan** [comment]
I am fine with this. Not crazy about the -dev -rc1 stuff since this can cause rpm package versioning to get screwed up.

In Rawhide if we shipped these, the rc1 would take precedence over the -dev.  Which might not be what you expect.  We could just bounce the sublevel.  But this is what docker does, and we end up truncating off the -dev field.

container-storage-setup-1.0.0-dev

contianer-storage-setup-1.0.0-rc1

**Comment by rhvgoyal** [comment]
I think -dev and -rc1 will be used for keeping track of status upstream and ideally rpms should not be packaging it. 

Once we are ready to release something for packaging, then we can get rid of -dev and -rc1 and packagers can take that version and package it.

**Comment by rhvgoyal** [comment]
@shishir-a412ed can you update this PR. We probably need global variables as I describe in my comments.

**Comment by rhatdan** [comment]
We have gotten this from another pull request.

**Pull request closed without merging**